### DISCUSSION TITLE
Build Failed on Fedora 39

### DISCUSSION THREAD

**Discussion by luizsouzadev** [rbenv/ruby-build#2368] [Build failures]
rbenv install 3.6.5

||||||||||||||||||||||||||||||||||||||||||||||||||||

BUILD FAILED (Fedora Linux 39 using ruby-build 20221004)

Inspect or clean up the working tree at /tmp/ruby-build.20240422161622.307563.z7FmRw
Results logged to /tmp/ruby-build.20240422161622.307563.log

Last 10 log lines:
To eliminate this warning, please install libyaml and reinstall your ruby.
The Ruby yaml extension was not compiled.
ERROR: Ruby install aborted due to missing extensions
Try running `yum install -y libyaml-devel` to fetch missing dependencies.

Configure options used:
  --prefix=/home/luizdev/.rbenv/versions/2.3.2
  --with-openssl-dir=/home/luizdev/.rbenv/versions/2.3.2/openssl
  LDFLAGS=-L/home/luizdev/.rbenv/versions/2.3.2/lib 
  CPPFLAGS=-I/home/luizdev/.rbenv/versions/2.3.2/include

**Comment by mislav** [comment]
Hi, Ruby version "3.6.5" does not exist, so I'm not sure which version you are trying to install. Also, your version of ruby-build is several years out of date.

You could try upgrading ruby-build to the latest version and then installing the latest version of Ruby:
```
git clone https://github.com/rbenv/ruby-build.git "$(rbenv root)/plugins/ruby-build"
rbenv install 3.3.1
```
